lib/generators/provider/templates/spec/spec_helper.rb

```diff
@@ -3,11 +3,10 @@
   SimpleCov.start
 end
 
-# Uncomment in case you use vcr cassettes
-# VCR.configure do |config|
-#   config.ignore_hosts 'codeclimate.com' if ENV['CI']
-#   config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
-# end
+VCR.configure do |config|
+  config.ignore_hosts 'codeclimate.com' if ENV['CI']
+  config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
+end
```

Comment:
I don't like this being the default.  It feels like it should be opt-in.  Can we make it a generator option?